README.md

```diff
@@ -0,0 +1,71 @@
+# Get An Identity
+
+A service that allows trainee teachers, applicants, praticipants and teachers to get an identity that they can use across
+DfE Teacher Services digital services and allows the DfE to provide better user experiences by joining user data across the services
+to prevent re-keying details.
+
+## Environments
+Information coming soon...
+
+### Useful Links
+
+| Name           | URL                                                                                                            |
+| -------------- | -------------------------------------------------------------------------------------------------------------- |
+| Design History | [get-an-identity](https://tra-digital-design-history.herokuapp.com/get-an-identity/)                 |
+| Tech. Arch.    | [high-level-technical-design](/docs/images/get-an-identity-v-hld.jpg)
+| ADR'S          | [technical-decisions](/docs/architecture/decisions/) |
+| Tech Docs      | [tech-docs-site](https://teacher-services-tech-docs.london.cloudapps.digital/)
+| TRN Design     | [trn-design-history](https://tra-digital-design-history.herokuapp.com/teacher-self-service-portal/trn-holders/)
+### Details and configuration
+
+| Name           | Description                                   | Status
+| -------------- | --------------------------------------------- | -------------- |
+| Production     | Public site                                   | Coming Soon    |
+| Pre-production | For internal use by DfE to test deploys       | Coming Soon    |
+| Test           | For external use to test                      | Coming Soon    |
+| Development    | For internal use by DfE for dev.              | Coming Soon    |
+
+## Dependencies
+
+- TBC
+
+## How the application works
+
+The application will provide an authentication server for teacher services digital applications (clients) to use in order to provide
+an "account" style function (where needed), authentication and authorisation flows based on OIDC/OATH 2.0 protocols. It will define
```

Comment:
nit:
```suggestion
an "account" style function (where needed), authentication and authorisation flows based on OIDC/OAUTH 2.0 protocols. It will define
```